src/main/java/bio/terra/service/tabulardata/google/BigQueryPdao.java

```diff
@@ -1953,16 +1953,7 @@ public void validateDeleteRequest(
     }
   }
 
-  /*
-   * WARNING: Ensure SQL is validated before executing this method!
-   */
-  public List<Map<String, Object>> getSnapshotTableData(Snapshot snapshot, String sql)
-      throws InterruptedException {
-    // execute query and get result
-    final BigQueryProject bigQueryProject = BigQueryProject.from(snapshot);
-    final TableResult result = bigQueryProject.query(sql);
-
-    // aggregate into single object
+  public List<Map<String, Object>> aggregateTableColumnsRows(TableResult result) {
```

Comment:
It looks like this method can be `static`. Also it can be `private` since it's only used in this class.
```suggestion
  private static List<Map<String, Object>> aggregateTableColumnsRows(TableResult result) {
```

Reply:
Whoops yeah. I saw the duplicated code and just pulled it out.